Core/include/Acts/Clusterization/Clusterization.hpp

```diff
@@ -8,6 +8,10 @@
 
 #pragma once
 
+#pragma message(                         \
```

Comment:
Do we actually have a client that depends on that?

Reply:
Not sure, probably not. See discussion in https://github.com/acts-project/acts/issues/1219. Would you prefer we remove it wholesale instead?

Reply:
Deleted the files, as discussed in the last dev meeting